### DISCUSSION TITLE
Error: getaddrinfo ENOENT raw. githubusercontent com Uncaught Error: getaddrinfo ENOENT raw.aithubusercontent com

### DISCUSSION THREAD

**Discussion by balms-web** [GitSquared/edex-ui#1048] [Questions]
![2021Y02M18D_123536](https://user-images.githubusercontent.com/63351589/108305848-e0544280-71e5-11eb-8974-4689f874421a.png)

**Comment by wrac4242** [comment]
hi, can you make an issue please, and give some more info on OS, and which version you are running

**Reply by GitSquared** [reply]
Looks like a case for #1050
